01 - Community/Obsidian Roundup/2022-01-01  Recipe Importing, Linking to Aliases, and Semantic Search.md

```diff
@@ -0,0 +1,138 @@
+-   [Plaintext](https://github.com/dbarenholz/obsidian-plaintext) by `@dbarenholz`  lets users open specified files as plaintext (RAW mode).
+-   [Text-to-Speech Plugin](https://github.com/brainoverflow98/obsidian-text-to-speech-plugin) by `@brainoverflow98`  is a simple text-to-speech plugin which uses the Chromium's SpeechSynthesis API.
+
+### Beta
+
+_Note: these plugins have not yet been submitted for code review, and are being made available primarily for testing purposes._
+
+-   The [cooklang importer](https://github.com/nothingislost/obsidian-cooklang-importer) brings us one step closer to Obsidian becoming a fully featured recipe manager. The developer hopes to one day improve the parser and merge the functionality into the "main" [cooklang plugin](https://github.com/deathau/cooklang-obsidian), but I've been using it. It's definitely an improvement over importing manually. I look forward to our community & the cooklang community one day sharing recipes between us in an open protocol sort of way. I _personally_ feel that there's huge potential here to compete with more expensive recipe (& shopping list) databases like plan2eat and paprika.
+-   You can also do plain text accounting thanks to the [Ledger plugin](https://github.com/tgrosinger/ledger-obsidian), which just entered early beta.
+-   You can now [import a JSON file and handlebars template](https://github.com/farling42/obsidian-import-json) to create a series of notes inside a named folder in your vault.
+-   [Coalescer](https://github.com/geoffreysflaminglasersword/Coalescer) allows you to easily merge files, links, and tags with each other to better organize your vault.
+-   [Remotely Save](https://github.com/fyears/remotely-save) supports mobile (especially iOS) to share notes between local device and S3, Dropbox, webdav, (OneDrive coming soon).
+-   You can combine [Templater](https://discord.com/channels/686053708261228577/875720842443649045/926258548650942565)  and the new [Auto Linker](https://github.com/nothingislost/obsidian-auto-linker) plugin to have all of your filenames be prefixed with timestamps but all of your wikilinks be formatted with just the `[[words from the title]]` because it will link directly to the alias (which templater can create). I just started using it so that I can link to `[[2021.02.01 Bugs are cool]]` using a  programmatically created alias, `[[Bugs are cool newsletter]]` for example, and there are other similar uses-cases folks might find handy as long as you don't tend to use the same alias for multiple files. I'm personally finding this _very_ useful when I'm annotating things in non-obsidian programs and don't have access to the quick switcher but do want to "link" to a particular file I remember the name of but not the number of... but am relying on numbers in the filename for sorting. Here's a [demo](http://discordapp.com/channels/686053708261228577/707816848615407697/920527336904986625) and further discussion about this.
```

Comment:
```suggestion
-   You can combine [Templater](https://discord.com/channels/686053708261228577/875720842443649045/926258548650942565)  and the new [Auto Linker](https://github.com/nothingislost/obsidian-auto-linker) plugin to have all of your filenames be prefixed with timestamps but all of your wikilinks be formatted with just the `[[words from the title]]` because it will link directly to the alias (which templater can create). I just started using it so that I can link to `[[2021.02.01 Bugs are cool]]` using a programmatically created alias, `[[Bugs are cool newsletter]]` for example, and there are other similar uses-cases folks might find handy as long as you don't tend to use the same alias for multiple files. I'm personally finding this _very_ useful when I'm annotating things in non-obsidian programs and don't have access to the quick switcher but do want to "link" to a particular file I remember the name of but not the number of... but am relying on numbers in the filename for sorting. Here's a [demo](http://discordapp.com/channels/686053708261228577/707816848615407697/920527336904986625) and further discussion about this.
```

Reply:
Suggested edit: remove the double-space in the middle of this bullet, as at some window widths it's causing jagged indentation